packages/config/test/methods.test.ts

```diff
@@ -38,6 +38,28 @@ describe("TruffleConfig.detect", () => {
   });
 });
 
+describe(".load", () => {
+  const config1 = `${__dirname}/testConfig.js`;
+  const config2 = `${__dirname}/testConfig2.js`; // identical to config1
+  const config3 = `${__dirname}/testConfig3.js`; // different mnemonic from 1&2
+
+  it("providers should not be ===", () => {
```

Comment:
@davidmurdoch I'm seeing the same results for this on `develop` and on this branch.

Reply:
I thought provider was a getter, not a function, per https://github.com/trufflesuite/truffle/pull/5029/files#diff-180639c74cbd9b1cd8f6bd257ccdfd81d6ea56545c59f9e93fb76f3225f661d2R288-R301